daffodil-runtime1/src/main/scala/edu/illinois/ncsa/daffodil/processors/unparsers/Unparser.scala

```diff
@@ -116,6 +120,7 @@ trait SuspendableUnparser
   protected def suspendableOperation: SuspendableOperation
 
   override final def unparse(state: UState): Unit = {
+    state.bitOrder
```

Comment:
I assume this is called so that checkBitOrder is called? Maybe add a comment to that effect?

Reply:
Added.